### PR TITLE
WIP: Makefile.generates-files: fix package typo

### PR DESCRIPTION
**Pull request by sttts**
In `Makefile.generated_scripts` we call kube-openapi's openapi-gen 
1. to create the OpenAPI spec for (a) kube-apiserver (b) aggregation-apiserver and (c) sample-apiserver
2. to derive api violations for the processed type (like a linter).

The script had a typo such that the `k8s.io/apiextensions-apiserver/pkg/apis/apiextensions ` were not included. For the violations this means that we just did not check whether the CRD types follow the common rules. Note, that these violations are not critical, but must be "accepted" by commiting them into the violation files under `api/api-rules`.

Moreover, apiextensions-apiserver's served OpenAPI v2 spec lacked the CRD types.

```release-note
The OpenAPI v2 spec served at `/openapi/v2` by kube-apiserver lacked the `apiextensions.k8s.io` API group types.
```

### CODE REVIEW

**Review by justaugustus — CHANGES_REQUESTED** [review]
@sttts -- This mentions a typo fix, but resulted in a change to `api/api-rules/apiextensions_violation_exceptions.list`.

Can you provide some details in the commit messages, PR title, and PR description about how/why this was done + a link to issue (if there's an existing one)?
Maybe a release note too, if it's user-facing?

/hold

**Comment by cblecker** [comment]
/approve
Approving for build. You'll need to actually regenerate things too, so that verify passes.

To be clear for api reviewers, there is no `k8s.io/apiextensions` so this fixes the reference to `k8s.io/apiextensions-apiserver` which is an actual repo :)

**Comment by justaugustus** [comment]
Thanks for updating the description!
/hold cancel
/retest

**Comment by cblecker** [comment]
/lgtm
/approve
/assign @lavalamp @liggitt 
for API approval

File: api/api-rules/apiextensions_violation_exceptions.list [hunk 1]
```diff
@@ -27,6 +68,37 @@ API rule violation: list_type_missing,k8s.io/apimachinery/pkg/apis/meta/v1,Table
...
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaProps,XMapType
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaProps,XPreserveUnknownFields
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrArray,JSONSchemas
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrArray,Schema
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrBool,Allows
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrBool,Schema
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrStringArray,Property
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1,JSONSchemaPropsOrStringArray,Schema
+API rule violation: names_match,k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1,CustomResourceColumnDefinition,JSONPath
```
Comment (liggitt) [inline api/api-rules/apiextensions_violation_exceptions.list]: this is the only warning we would have actually wanted to know about, and we already found/fixed it in v1 (https://github.com/kubernetes/kubernetes/issues/66531)

**Comment by justaugustus** [comment]
@sttts -- Needs rebase

**Pull request closed without merging**